tests/stepDefinitions/loginContext.js

```diff
@@ -1,35 +1,39 @@
 const { Given, When, Then } = require('@cucumber/cucumber')
 const { expect } = require("@playwright/test");
-
-
-// for valid scenario
-Given('user {string} has navigated to the admin login page', function (string) {
-  console.log(" Given ");
+const loginBtnLocator=".btn.btn-block.btn-primary"
+const emailFieldLocator="#spree_user_email"
+const passwordFieldLocator="#spree_user_password"
+const dashboardSelector="//*[@id='missingHomepage']/div/div[2]/h4[1]"
```

Comment:
If you want to go for that `h1`, then you can simply use: `//*[@id='missingHomepage']//div//h4[1]`
or
we can use this &darr;
const dashboardSelector = `//div[@id='missingHomepage']//h1`